adafruit_platformdetect/board.py

```diff
@@ -114,7 +114,11 @@ def id(self):
         elif chip_id == chips.HFU540:
             board_id = self._sifive_id()
         elif chip_id == chips.C906:
+            board_id = self.__beaglebone_id()
+        elif chip_id == chips.VICU7:
             board_id = self._allwinner_id()
+        elif chip_id == chips.VICU7:
+            board_id = self.__beaglebone_id()
```

Comment:
It appears the second chip_id == chips.VICU7 will never be reached. I'm guessing the first one was intended to be removed.